$OpenBSD: patch-lib_CMakeLists_txt,v 1.23 2019/07/15 21:45:43 sebastia Exp $

- Override library version
- only include utmpx_file when there is utmpx.h on the system

Index: lib/CMakeLists.txt
--- lib/CMakeLists.txt.orig
+++ lib/CMakeLists.txt
@@ -330,6 +333,7 @@ set_target_properties(libfactersrc PROPERTIES POSITION
 add_library(libfacter $<TARGET_OBJECTS:libfactersrc>)
 if (BUILD_SHARED_LIBS)
     set_target_properties(libfacter PROPERTIES PREFIX "" SUFFIX ".so" IMPORT_PREFIX "" IMPORT_SUFFIX ".so.a" VERSION ${PROJECT_VERSION})
+    set_target_properties(libfacter PROPERTIES PREFIX "" SUFFIX ".so" IMPORT_PREFIX "" IMPORT_SUFFIX ".so.a" VERSION ${LIBfacter_VERSION})
 endif()
 
 if(AIX)
